addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -87,15 +91,13 @@
                             onOptionsChange.bind="onSelectionOptionChange"
                             onDefaultOptionChange.bind="onDefaultChange"
                         />
-                    </td>
-                </tr>
-                <tr t-if="state.propertyDefinition.type === 'tags'">
-                    <td class="o_td_label align-top">
-                        <label t-att-for="getUniqueDomID('tags')">
-                            <b>Tags</b>
-                        </label>
-                    </td>
-                    <td>
+                    </div>
+                </div>
+                <div t-if="state.propertyDefinition.type === 'tags'" class="d-contents mb-3 mb-sm-0">
```

Comment:
Same as before: `text-900`applied on labels and components directly + remove the`<div>`'s (if possible).
Could you check for other occurrences using the same structure too? Like `DomainSelector` etc..